code/drasil-database/lib/Database/Drasil/Dump.hs

```diff
@@ -28,5 +28,4 @@ dumpChunkDB cdb =
     $ insert "theoryModels" (umapDump $ cdb ^. theoryModelTable)
     $ insert "conceptInstances" (umapDump $ cdb ^. conceptinsTable)
     $ insert "labelledContent" (umapDump $ cdb ^. labelledcontentTable)
-    $ insert "references" (umapDump $ cdb ^. refTable)
```

Comment:
This is the effective "demotion" of references to "not a chunk" that removes it from consideration in the problematic chunks.json file generation.